jpype/_core.py

```diff
@@ -222,6 +222,10 @@ def startJVM(*args, **kwargs):
                     jvmpath, version)) from ex
         raise
 
+
+def initializeResources():
+    global _JVM_started
+    _JVM_started = True
```

Comment:
I'd set this state after all resources had been initialized, e.g. if something goes wrong, the state is not correct.

Reply:
Sounds like a plan.